scripts/scil_compute_rish_from_sh.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+Script to compute the RISH features from an SH coefficients image.
+"""
+import argparse
+
+import nibabel as nib
+import numpy as np
+
+from scilpy.io.utils import add_overwrite_arg, assert_inputs_exist, \
+    assert_outputs_exist
+from scilpy.reconst.sh import compute_rish
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(
+        description=__doc__, formatter_class=argparse.RawTextHelpFormatter)
+    p.add_argument('in_sh',
+                   help='Path of the sh image.')
```

Comment:
Does it support all SH files?
- Only symmetric SH? Assymetric SH too? (would be cool to support any SH basis)
- Does it support all orders?